client/src/features/projectsV2/fields/SlugFormField.tsx

```diff
@@ -70,10 +70,13 @@ export default function SlugFormField<T extends FieldValues>({
       }}
       rules={{
         required: true,
-        maxLength: 99,
+        maxLength: {
+          value: 99,
+          message: "This slug exceed maxLength.",
```

Comment:
This does not sound useful for users.

```suggestion
          message: "The slug must not exceed 99 characters.",
```